### PR TITLE
chore: fix wrong boolean exp

### PR DESCRIPTION
**Pull request by IvanJobs**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/17292555

### CODE REVIEW

**Review by hund030 — APPROVED** [review]

File: packages/fx-core/tests/component/driver/botAadApp/create.test.ts [hunk 1]
```diff
@@ -184,4 +184,31 @@ describe("botAadAppCreate", async () => {
...
     );
   });
+
+  it("should be good when reusing existing bot in env", async () => {
+    const originalValues = {
+      BOT_ID: process.env.BOT_ID,
+      SECRET_BOT_PASSWORD: process.env.SECRET_BOT_PASSWORD,
+    };
+    // mock env
+    process.env.BOT_ID = expectedClientId;
```
Comment (hund030) [inline packages/fx-core/tests/component/driver/botAadApp/create.test.ts:194]: It's better to use mockedEnv

**Review by hund030 — DISMISSED** [review]

File: packages/fx-core/tests/component/driver/botAadApp/create.test.ts (hunk 1 above)
Comment (IvanJobs) [reply packages/fx-core/tests/component/driver/botAadApp/create.test.ts]: Good suggestion! It's the first time for me to use `mockedEnv`.

File: packages/fx-core/tests/component/driver/botAadApp/create.test.ts [hunk 2]
```diff
@@ -184,4 +184,25 @@ describe("botAadAppCreate", async () => {
...
+
+    const result = await createBotAadAppDriver.execute(args, mockedDriverContext);
+
+    expect(result.result.isOk()).to.be.true;
+    expect(result.result.isOk() && result.result.value.get(outputKeys.BOT_ID)).to.be.equal(
+      expectedClientId
+    );
+    expect(
+      result.result.isOk() && result.result.value.get(outputKeys.SECRET_BOT_PASSWORD)
+    ).to.be.equal(expectedSecretText);
```
Comment (hund030) [inline packages/fx-core/tests/component/driver/botAadApp/create.test.ts:206]: You will need to call `envRestore()` after the test.

Comment (IvanJobs) [reply packages/fx-core/tests/component/driver/botAadApp/create.test.ts:206]: `envRestore()` will be called in `afterEach()`.

**Review by hund030 — APPROVED** [review]

**Pull request merged**